common/rabbitmq/templates/etc/_definitions.json.tpl

```diff
@@ -13,14 +13,28 @@
     "users": [
 {{- range $_key, $_user := merge .Values.users }}
 {{- $tags := "" }}
-{{- $users := dict "name" $_user.user "tags" (tuple $_user.user | include "rabbitmq.tags") "password" (tuple $envAll $_user.user $_user.password | include "rabbitmq.pass") }}
+{{- $users := dict "name" $_user.user "tags" (tuple $_user.user | include "rabbitmq_tags") "password" (tuple $envAll $_user.user $_user.password | include "rabbitmq_pass") }}
 {{ toJson $users | indent 8 }},
 {{- end }}
 {{- if .Values.metrics.enabled }}
-        {"name":"{{.Values.metrics.user}}","password":"{{tuple . .Values.metrics.user .Values.metrics.password | include "rabbitmq.pass"}}","tags":"monitoring"},
+        {"name":"{{.Values.metrics.user}}","password":"{{tuple . .Values.metrics.user .Values.metrics.password | include "rabbitmq_pass"}}","tags":"monitoring"},
 {{- end }}
-        {"name":"guest","password":"{{ tuple . .Values.users.default.user .Values.users.default.password | include "rabbitmq.pass" }}","tags":"monitoring"}
+        {"name":"guest","password":"{{ tuple . .Values.users.default.user .Values.users.default.password | include "rabbitmq_pass" }}","tags":"monitoring"}
     ],
+{{- if .Values.cluster.ha }}
```

Comment:
I would prefer if we can set the policy via the values more fine grained. In particular, for RPC it might sense to set a TTL on the messages.